products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp

```diff
@@ -15,6 +15,17 @@
 namespace
 {
     static constexpr std::chrono::milliseconds betweenSessionDelayTime{5000};
```

Comment:
```suggestion
    constexpr std::chrono::milliseconds betweenSessionDelayTime{5000};
```

No need for `static` keyword, all of these values are already in anonymous namespace which serves the same purpose